agent/fw_laminas3.c

```diff
@@ -153,4 +153,5 @@ void nr_laminas3_enable(TSRMLS_D) {
   nr_php_wrap_user_function(
       NR_PSTR("Laminas\\Mvc\\Console\\Router\\RouteMatch::setMatchedRouteName"),
       nr_laminas3_name_the_wt TSRMLS_CC);
+  nr_txn_add_php_package(NRPRG(txn), "laminas/laminas-mvc", " ");
```

Comment:
What is the reasoning behind the last parameter being a space `" "` as opposed to, say, empty `""`?

Reply:
In the `update_loaded_modules` spec, it gives an example of how the json should look without a version and it passes in the version as an empty string with a space like `" "`, which is why we are following this format.

Reply:
I'm still thinking it would be good to have a `#define PHP_PACKAGE_VERSION_UNKNOWN ""` or similar so we don't have these odd string constants sprinkled all over the code.

Reply:
Good idea, fixed: [71f9f38](https://github.com/newrelic/newrelic-php-agent/pull/775/commits/71f9f389af9281373537a8eba7c23e727466188f)